.changes/nextrelease

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
Just noticed this after merging in the RDS waiter PR, the changelog file needs to be in a named file inside a nextrelease folder:

>A changelog document is a small JSON blob placed in the .changes/nextrelease folder. It should be named clearly and uniquely, akin to a branch name.